script.js

```diff
@@ -0,0 +1,88 @@
+// const fs = require('fs');
+
```

Comment:
Consider looking into namespacing an App object to keep all your code organized. Also, document readys are your friend with a handy init method. :)

Reply:
I'm not sure what document readys are but that definitely clues me in to something I should learn :)